packages/subgraph/schema.graphql

```diff
@@ -107,70 +107,107 @@ type Dao @entity {
   balances: [Balance!] @derivedFrom(field: "dao")
   contractPermissionIds: [ContractPermissionId!]! @derivedFrom(field: "dao")
   permissions: [Permission!]! @derivedFrom(field: "dao")
-  plugins: [DaoPlugin!] @derivedFrom(field: "dao")
+  pluginInstallations: [PluginInstallation!]! @derivedFrom(field: "dao")
+  plugins: [IPluginInstallation!]! @derivedFrom(field: "dao")
```

Comment:
are going to remove `pluginInstallations` or `plugins` attribute in the future ? once plugins are out of the core? if yes, i would add a TODO comment here...etc

Reply:
good point. added and we merge